app/models/metadata_field.rb

```diff
@@ -130,6 +137,17 @@ def metadata_field_subsets
     if is_required == 1 && is_default == 0
       errors[:name] << 'Required field must also be default field'
     end
+    if is_required == 1 && !default_for_host_genome?
+      errors[:name] << 'Required field must also be default_for_new_host_genome field'
+    end
+  end
+
+  # This method is designed to be called when a field is made required or
```

Comment:
Nit: This comment looks out of date, as this method is currently being called whenever a field is updated.